server.go

```diff
@@ -386,6 +386,46 @@ func (server *Server) createTLSConfig(entryPointName string, tlsOption *TLS, rou
 	// BuildNameToCertificate parses the CommonName and SubjectAlternateName fields
 	// in each certificate and populates the config.NameToCertificate map.
 	config.BuildNameToCertificate()
+	//Set the minimum TLS version if set in the config TOML
+	if server.globalConfiguration.EntryPoints[entryPointName].TLS.MinVersion != "" {
+		config.PreferServerCipherSuites = true
+		switch server.globalConfiguration.EntryPoints[entryPointName].TLS.MinVersion {
+		case "VersionTLS10":
+			config.MinVersion = tls.VersionTLS10
+		case "VersionTLS11":
+			config.MinVersion = tls.VersionTLS11
+		case "VersionTLS12":
+			config.MinVersion = tls.VersionTLS12
+		}
+	}
+	//Set the list of CipherSuites if set in the config TOML
+	if server.globalConfiguration.EntryPoints[entryPointName].TLS.CipherSuites != nil {
```

Comment:
This also allows for an easy pass on compatibility tests. If not defined, do nothing.